chain/blockchain.go

```diff
@@ -31,7 +32,7 @@ func NewBlockchainWithGenesisBlock(store ILedgerStore) (*Blockchain, error) {
 		return nil, err
 	}
 
-	root, err := store.GenerateStateRoot(genesisBlock, false, false)
+	root, err := store.GenerateStateRoot(context.Background(), genesisBlock, false, false)
```

Comment:
how to trigger it?

Reply:
GenerateStateRoot will not timeout when persist or create genesis block, but will timeout during consensus at https://github.com/nknorg/nkn/blob/master/consensus/proposal.go#L131

Reply:
ok, I see